### DISCUSSION TITLE
Convert GraphicsPath to SVGPath

### DISCUSSION THREAD

**Discussion by Perissinotto** [vvvv/SVG#850] [Q&A]
Is it possible to Convert graphicsPath to SvgPath to write it in a file?

Thanks for helping

**Comment by mrbean-bremen** [comment]
To my knowledge, there is no such functionality.
I think you have to get the `PathData` from `GraphicsPath`, iterate over the points, create `SvgPathSegment` objects from them, add them to a new ` SvgPathSegmentList`, and set that in a new `SvgPath`.
Basically, do it yourself manually...
Maybe someone else knows of a faster way.
